sample-app/src/main/java/com/joinforage/android/example/ui/pos/screens/refund/RefundResultScreen.kt

```diff
@@ -45,7 +49,20 @@ fun RefundResultScreen(
             horizontalAlignment = Alignment.CenterHorizontally
         ) {
             if (txType == null) {
-                Text("null txType")
+                Text("Unable to determine transaction type. Terminal might be offline.")
+                if (fetchedPayment?.ref == null) {
+                    Text("Re-fetch payment to see a list of refunds on the payment.")
+                    Button(onClick = onReloadButtonClicked) {
+                        Text("Re-fetch Payment")
+                    }
+                } else {
+                    Text("Select a Refund ref from payment (${fetchedPayment.ref}) to view the receipt for:")
+                    fetchedPayment.refunds.forEach { refundRef ->
+                        Button(onClick = { onRefundRefClicked(fetchedPayment.ref!!, refundRef) }) {
+                            Text(refundRef)
+                        }
+                    }
```

Comment:
I cannot believe that making a list of `Button`s is as easy as doing a `forEach` loop 🤯🤯🤯🤯. Compose is soooo effing cool!

Reply:
Right?! I had to look it up and was like "oh wow that's super easy"